src/main/java/hng_java_boilerplate/user/dto/request/SignupDto.java

```diff
@@ -0,0 +1,29 @@
+package hng_java_boilerplate.user.dto.request;
+
+import hng_java_boilerplate.util.ValidPassword;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class SignupDto {
+    @NotBlank(message = "Full name is required")
```

Comment:
In line 15, you mean to say First name is required.

Reply:
@Olaoluwa22 Done